recipes/rvlib/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "rvlib" %}
+{% set version = "0.0.1" %}
+{% set sha1 = "40110f5cf76a0fa29fa979179d66126170bf4d58" %} 
+
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/QuantEcon/rvlib/archive/{{ version }}.tar.gz
+  sha1: {{ sha1 }}
+
+build:
+  skip: True  # [win or not py35]
+  number: 2
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python  # [py35]
+    - setuptools
+    - numpy
+    - cffi
+    - numba
+    - pyyaml
+
+  run:
+    - python  # [py35]
+    - numpy
+    - cffi
+    - numba
+    - pyyaml
+
+
+test:
+  imports:
+    - rvlib
+
+about:
+  home: https://github.com/QuantEcon/rvlib.git
+  license: BSD-3
+  summary: 'Mimicking the API of Distributions.jl in Python with added numba support.'
+  description: |
+    rvlib is a statistical package that mimics the API of Julia's 
+    Distributions.jl package while at the same time achieves a similar 
+    performance by exploiting numba.
+  dev_url: https://github.com/QuantEcon/rvlib
+
+extra:
+  recipe-maintainers:
+    - danielcsaba
+    - spencerlyon2
```

Comment:
Are you ok with being added as a maintainer of `rvlib` at [conda-forge]( https://conda-forge.github.io ), @spencerlyon2?